keps/sig-node/2570-memory-qos/README.md

```diff
@@ -104,6 +107,145 @@ Brief map as follows:
+
+
+The table below runs over the examples with different values requests.memory:
+
+| limits.memory (1000) | memory throttling factor (0.9)|
+| ---------------------- | ----------------------------- |
+| request 0    | 900  |
+| request 100  | 910  |
+| request 200  | 920  |
+| request 300  | 930  |
+| request 400  | 940  |
+| request 500  | 950  |
+| request 600  | 960  |
+| request 700  | 970  |
+| request 800  | 960  |
+| request 900  | 980  |
+| request 1000 | 1000 |
+
+
+Node reserved resources(kube-reserved/system-reserved) are either considered. It is tied to `--enforce-node-allocatable` and `memory.min` will be set properly.
+
+Brief map as follows:
+| type | memory.min | memory.high |
+| -------- | -------- | -------- | 
+| container | requests.memory | floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize |
+| pod | sum(requests.memory) | n |
+| node | n/a | pods, kube-reserved, system-reserved | n |
+
+###### Reasons for changing the formula of memory.high calculation in Beta
+
+The formula for memory.high has changed in Beta as the Alpha implementation has following problems:
+1. It fails to throttle when requested memory is closer to memory limits (or node allocatable): memory.high < requests.memory. e.g. If `requests.memory = 85, limits.memory=100`, and we have a throttling factor of 0.8, then as per the Alpha implementation memory.high =  memory throttling factor * limits.memory i.e. memory.high = 80. In this case the level at which throttling is supposed to occur i.e. memory.high is less than requests.memory. Hence there won't be any throttling as the Alpha implementation doesn't allow memory.high to be less than requested memory. 
+
+2. It could result in early throttling even when a sizeable chunk of memory is unused. 
+  
+    For example, 
+    * `requests.memory` = 800Mi
+
+      `memory throttling factor` = 0.8
+
+      `limits.memory` = 1000Mi
+
+      As per Alpha implementation,
+
+      `memory.high` = memory throttling factor * limits.memory  = 0.8 * 1000Mi = 800Mi
+
+      Throttling occurs at 800Mi, while a sizable chunk of 200Mi is still unused.
+
+    * `requests.memory` = 500Mi
+
+      `memory throttling factor` = 0.6
+
+      `limits.memory` = 1000Mi
+      
+      As per Alpha implementation,
+
+      `memory.high` = memory throttling factor * limits.memory = 0.6 * 1000Mi = 600Mi
+      
+      Throttling occurs at 600Mi which is just a 100Mi over the requested memory. 400Mi of memory is still unused as early throttling happens.
+  
+
+3. Default throttling factor of 0.8 might be too aggressive for some applications that are latency sensitive and always use memory close to memory limits.
+
+   For example, there are some known Java workloads that use 85% of the memory will start to get throttled once this feature is enabled by default. Hence the default 0.8 MemoryThrottlingFactor value may not be a good value.
```

Comment:
s/may not be a good value/may not be a good value for many applications due to inducing throttling too early.